Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/DriverOperatorAssociation.yaml

```diff
@@ -0,0 +1,39 @@
+imports:
+  Merchant: Domain.Types.Merchant
+  MerchantOperatingCity: Domain.Types.MerchantOperatingCity
+  Person: Domain.Types.Person
+
+DriverOperatorAssociation:
+  tableName: driver_operator_association
+
+  fields:
+    id: Id DriverOperatorAssociation
+    operatorId: Text
+    isActive: Bool
+    driverId: Id Person
+    associatedOn: Maybe UTCTime
+    associatedTill: Maybe UTCTime
+    createdAt: UTCTime
+    updatedAt: UTCTime
+
+  constraints:
+    id: PrimaryKey
+    operatorId: SecondaryKey
+    driverId: SecondaryKey
+
+  queries:
+    findAllOperatorByDriverId:
+      kvFunction: findAllWithKV
+      where: driverId
+
+    deleteByOperatorId:
+      kvFunction: deleteWithKV
+      where: operatorId
+
+  sqlType:
+    id: text
+    operatorId: text
+    driverId: text
+
+
+  excludedFields: [merchantOperatingCityId, merchantId, createdAt, updatedAt]
```

Comment:
you can remove createdAt and updatedAt from lines 16-17 and here from excluded list
These fields will be added automatically